uwsgi_tasks/tasks.py

```diff
@@ -230,6 +238,14 @@ def create(cls, **kwargs):
 class RuntimeTask(BaseTask):
     executor = TaskExecutor.RUNTIME
 
+    def execute_now(self):
+        try:
+            _serialize(self.args)
+            _serialize(self.kwargs)
+        except AttributeError as ex:
```

Comment:
I don't think we need to catch any exception here (especially AttributeError?), since if arguments are not pickable, PicklingError is generated so the problem is detected.